libs/components/src/lib/file-picker/file-picker.template.ts

```diff
@@ -18,7 +18,7 @@ export const FilePickerTemplate = (context: VividElementDefinitionContext) => {
 			x.setButtonTag(context.tagFor(Button));
 			x.setIconTag(context.tagFor(Icon));
 		}}
-		<div class="base" aria-label="${(x) => x.label}">
+		<div class="base" aria-label="${(x) => x.ariaLabel}">
```

Comment:
Why add it at all? `aria-label` takes effect on the host anyway
```suggestion
		<div class="base">
```